pythontodo/src/todo_controller.py

```diff
@@ -0,0 +1,48 @@
+from flask import Flask, jsonify, request
+from todo_repository import TodoRepository
+
+app = Flask(__name__)
+
+repository = TodoRepository()
+
+@app.route('/todos', methods=['GET'])
+def get_all_todos():
+    todos = repository.get_all_todos()
+    return jsonify([todo.getJson() for todo in todos])
+
+@app.route('/todos', methods=['POST'])
+def create_todo():
+    data = request.get_json()
+    print(data)
+    todo = repository.create_todo(data['task'], data['description'],data['completed'], data['edited'],data['createdDate'], data['modifiedDate'],data['lastViewed'],data['serialNumber'])
```

Comment:
Cam we just pass `data` into the create_todo method?

Reply:
I will try if that work, not much familiar with the Python, just used it as per some documentation.